linter.py

```diff
@@ -8,7 +8,9 @@ class Flake8(PythonLinter):
 
     cmd = ('flake8', '--format', 'default', '${args}', '-')
     defaults = {
-        'selector': 'source.python'
+        'selector': 'source.python',
+        # By default, filter codes Sublime can auto-fix
+        'ignore-fixables': True
```

Comment:
- default: False
- document how to enable
- document which codes it ignores
- document which settings to enable in ST